line-bot-model/src/main/java/com/linecorp/bot/model/response/NarrowcastProgressResponse.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.bot.model.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+
+import com.linecorp.bot.model.response.NarrowcastProgressResponse.NarrowcastProgressResponseBuilder;
+
+import lombok.Builder;
+import lombok.Value;
+
+@Value
```

Comment:
Please add link to https://developers.line.biz/en/reference/messaging-api/#response-7

Reply:
Applied